src/ImageSharp.Web/Middleware/ResponseConstants.cs

```diff
@@ -26,17 +26,5 @@ internal static class ResponseConstants
         /// resource has been denied.
         /// </summary>
         internal const int Status412PreconditionFailed = 412;
-
-        /// <summary>
-        /// An empty completed task.
-        /// </summary>
-        internal static readonly Task CompletedTask = CreateCompletedTask();
-
-        private static Task CreateCompletedTask()
-        {
-            var tcs = new TaskCompletionSource<object>();
-            tcs.SetResult(null);
-            return tcs.Task;
-        }
```

Comment:
This seemed to be only referenced by `ImageContext.SendStatusAsync` which is now async.